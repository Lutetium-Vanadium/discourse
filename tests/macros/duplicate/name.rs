fn main() {
    let q = inquisition::questions![input {
        name: todo!(),
        name: todo!(),
    }];
}
