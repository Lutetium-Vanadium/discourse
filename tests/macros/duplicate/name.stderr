error: duplicate option `name`
 --> $DIR/name.rs:4:9
  |
4 |         name: todo!(),
  |         ^^^^
